docs/supported-technologies.asciidoc

```diff
@@ -86,6 +86,33 @@ We automatically instrument background processing using:
 - Sidekiq
 - Shoryuken
 - Sneakers (v2.12.0+) (Experimental, see https://github.com/elastic/apm-agent-ruby/pull/676[#676])
+- Resque (v2.0.0+)
+
+[float]
+[[supported-technologies-resque]]
+=== Resque
+
+You should require `elastic_apm/resque` in the Rakefile where the Resque rake tasks are required.
+Doing this will set up a hook to automatically start the ElasticAPM agent after Resque forks for the
+first time.
```

Comment:
```suggestion
```

I don't think we need to explain _why_ here.

Reply:
I think we should tell users that the agent will automatically start upon at the first fork though.

Reply:
We're already saying _Do this and Resque works_. I don't feel like we need to explain _why_ it does or _how_ it does so. It's all there in the code if anyone's curious.

Reply:
Also, they need to know that the agent will not automatically start if they don't include `elastic_apm/resque` in the `Rakefile`.

Reply:
The instructions are needed, not disagreeing.

```
To make the agent work with Resque, you need to require `elastic_apm/resque` before you boot your Resque worker process.

For example in your `Rakefile`:

# ... blah blah
```

I think something like that is sufficient. People don't notice the details anyway, so better get straight to the code example 😅

Reply:
ok👍🏼